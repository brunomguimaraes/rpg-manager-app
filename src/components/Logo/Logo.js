// @flow

import React from "react";
import { Link } from "react-router-dom";
import styles from "./Logo.module.scss";
import diceLogo from "~/assets/dice-logo.png";

export default function Logo() {
  return (
    <Link className={styles.logo} to="/">
      <img src={diceLogo} alt="Dice Logo" styles={{ marginRigth: "8px" }} />
      <strong>RPG Manager</strong>
    </Link>
  );
}
