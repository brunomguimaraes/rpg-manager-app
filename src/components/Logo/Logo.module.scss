@import "~styles/variables";

.logo {
  & > img {
    height: $spacer * 4;
    width: $spacer * 4;
    margin-right: $spacer;

    @media screen and (max-width: $break-small) {
      height: $spacer * 3;
      width: $spacer * 3;
    }
  }
}
