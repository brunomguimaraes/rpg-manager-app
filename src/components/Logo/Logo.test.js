import React from "react";
import Logo from "./Logo";

test("Logo", () => expect(shallow(<Logo />)).toMatchSnapshot());
