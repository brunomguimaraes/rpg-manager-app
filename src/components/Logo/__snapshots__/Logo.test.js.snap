// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<Link
  className="logo"
  replace={false}
  to="/"
>
  <img
    alt="Dice Logo"
    src="dice-logo.png"
    styles={
      Object {
        "marginRigth": "8px",
      }
    }
  />
  <strong>
    RPG Manager
  </strong>
</Link>
`;
